src/visualizations/MunicipalitiesChart.fs

```diff
@@ -122,11 +122,16 @@ let init (queryObj : obj) (data : RegionsData) : State * Cmd<Msg> =
                 |> Seq.sortBy (fun dp -> dp.Date)
                 |> Seq.toList
             let totalsShown = totals |> Seq.skip ((Seq.length totals) - showMaxBars) |> Seq.toList
+            let lastDay = Seq.tryLast dp
             let doublingTime =
+                if lastDay.IsSome && lastDay.Value.ActiveCases = Some(0) (* SLO-spec SLO does not have that if condition for ActiveCases == 0 *)
+                then None
+                else
```

Comment:
@lukarenko please check my F#-fu when you have the chance. I was trying to not calculate `doublingTime` for municipalities which have 0 Active Cases. This is the only way I managed to do it, but I am not sure it is the correct one :)

Thanks!